contract/src/main/java/io/neow3j/contract/NeoURI.java

```diff
@@ -159,14 +145,11 @@ public NeoURI toAddress(String address) {
     }
 
     public NeoURI toAddress(ScriptHash address) {
-        if (!AddressUtils.isValidAddress(address.toAddress())) {
-            throw new IllegalArgumentException("Invalid address used.");
-        }
-
         this.address = address;
         return this;
     }
 
+    // TODO: This method does the same as the second method below. What shoul done differently?
```

Comment:
Sorry for my bad TODO comment that I committed here 😬
The method `assetFromByteArray(String asset)` and `asset(String asset)` do the same thing but should probably behave differently. I guess the argument in `assetFromByteArray(String asset)` is expected to be scriptHash in hexadecimal from. But what do we expect in `asset(String asset)`?

Reply:
I changed this. Now the following methods exist: `asset(String asset)`, `asset(ScriptHash asset)` and `asset(byte[] asset)`.